### DISCUSSION TITLE
Allow reserved words as field handles

### DISCUSSION THREAD

**Discussion by a-rbsn** [craftcms/cms#16308] [Ideas] [answered]
I don't know if anyone else finds this, but when building the structure of a site in Craft, I encounter reserved words a lot — url, link, icon etc., and to get around this you have to come up with some arbitrary name for a field handle, when it'd be great to just make a link field and call it 'link', then refer to it as link in the front-end.

I understand why the reserved words are there from a "they're used in important places within craft" sense, but is there a way to allow them even if it's just faking it? Some way of separating the reserved words from handles?

**Comment by brandonkelly** [comment, marked as answer]
Thanks for posting that. The same point was brought up recently (#16109), so closing this as a duplicate.

**Reply by a-rbsn** [reply]
Ah sorry @brandonkelly — I did a search and didn't find anything else. Thank you.
